tests/js/utils.js

```diff
@@ -0,0 +1,237 @@
+
+import {Strip, strip, Cookies, cookies, DjangoAjax, PontoonDjangoAjax, ajax} from 'utils';
+
+
+test('PontoonDjangoAjax instance', () => {
+    expect(ajax instanceof DjangoAjax).toBe(true);
+    expect(ajax instanceof PontoonDjangoAjax).toBe(true);
+})
+
+
+test('DjangoAjax csrf', () => {
+    const _ajax = new DjangoAjax();
+    document.cookie = "csrf=tripe";
+    expect(_ajax.csrf).toBe('tripe');
```

Comment:
I'd try to check if:
* what you will get if there's no csrf cookie.
* there are multiple cookie keys.

Reply:
this is actually  a bit wrong - i think it should be `csrftoken`

i kinda added this for completeness, to make it clear what was different about our setup, and to have the code there if we switched to using django's csrf

Reply:
...ill add some comments etc...